linux_docker_resources/Dockerfile-RHEL

```diff
@@ -149,7 +148,7 @@ RUN chown rosbuild: /home/rosbuild/.colcon -R
 
 # Add an entry point which changes rosbuild's UID from 1234 to the UID of the invoking user.
 # This means that the generated files will have the same ownership as the host OS user.
-ADD entry_point.sh /entry_point.sh
+COPY entry_point.sh /entry_point.sh
```

Comment:
This change is a drive-by improvement since I removed then re-added this line.
[Docker's documentation](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy) recommends COPY over ADD unless you need ADD's specific features.

>  Although ADD and COPY are functionally similar, generally speaking, COPY is preferred. That’s because it’s more transparent than ADD. COPY only supports the basic copying of local files into the container, while ADD has some features (like local-only tar extraction and remote URL support) that are not immediately obvious. Consequently, the best use for ADD is local tar file auto-extraction...